### DISCUSSION TITLE
请问开始按钮在哪儿？

### DISCUSSION THREAD

**Discussion by UncleBlack-I** [mikumifa/biliTickerBuy#95] [提问专区]
第二页找不到开始按钮
![QQ截图20240611001030](https://github.com/mikumifa/biliTickerBuy/assets/172297375/9fd7a28d-ee98-44ad-a9e7-1e7454c5836e)

**Comment by mikumifa** [comment]
炸裂，加验证码不小心删掉了，先用老版

**Comment by mikumifa** [comment]
看一下最新版

**Reply by UncleBlack-I** [reply]
okk谢谢
